_api-reference/index-apis/refresh.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Refresh index
+parent: Index APIs
+nav_order: 61
+---
+
+# Refresh index 
+Introduced 1.0
+{: .label .label-purple }
+
+The refresh index API refreshes one or more indexes in an OpenSearch cluster. In the case of data streams, the refresh index API refreshes a stream's backing indexes. 
+
+OpenSearch's refresh behavior depends on whether or not the `index.refresh_interval` is set:
+
+- When set, indexes are refreshed based on the `index.refresh_interval` setting (in seconds).
+- When not set, refreshes occur every second until the shard receives no search requests for at least the number set in the `index.search.idle.after` setting (in seconds). Default is 30 s. 
+
```

Comment:
For more information about `refresh_interval`settings, see [Dynamic index-level index settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/index-settings/#dynamic-index-level-index-settings).